src/typings/react-native.d.ts

```diff
@@ -310,6 +317,11 @@ declare module 'react-native' {
         overScrollMode?: string; //enum( 'always', 'always-if-content-scrolls', 'never' )
         // iOS
         scrollIndicatorInsets?: {top: number, left: number, bottom: number, right: number };
+        // Windows only
+        onKeyDown? : Function;
+        onKeyUp? : Function;
+        tabNavigation? : 'local' | 'cycle' | 'once';
+        disableKeyboardBasedScrolling?: boolean;
```

Comment:
disableKeyboardBasedScrolling needs to be documented. Is it windows-specific? If so, it should also have a  comment.

Reply:
This is RN for Windows only, but we don't expose it over RX API.  I meant for the ""windows only" comment to apply to the whole paragraph, including disableKeyboardBasedScrolling. Where should I document more? Note this is react-native.d.ts, not Typings.ts


---
In reply to: [159786086](https://github.com/Microsoft/reactxp/pull/404#discussion_r159786086) [](ancestors = 159786086)